juniper_official/Linecard/pipe-stats.rule

```diff
@@ -0,0 +1,224 @@
+healthbot {
+    topic external {
+        rule pfe_pipe_stats {
+            synopsis "PFE Pipe stats";
+            description "PFE Pipe stats";
+            sensor pipestats {
+                open-config {
+                    sensor-name /components/component/integrated-circuit/pipeline-counters;
+                    frequency 60s;
+                }
+            }
+            field fabric-block-fabric-aggregate {
+                sensor pipestats {
+                    path /components/component/integrated-circuit/pipeline-counters/drop/fabric-block/state/fabric-aggregate;
+                }
+                type unsigned-integer;
+            }
+            field fabric-block-in-high-priority {
+                sensor pipestats {
+                    path /components/component/integrated-circuit/pipeline-counters/drop/fabric-block/state/in-high-priority;
+                }
+                type unsigned-integer;
+            }
+            field fabric-block-in-low-priority {
+                sensor pipestats {
+                    path /components/component/integrated-circuit/pipeline-counters/drop/fabric-block/state/in-low-priority;
+                }
+                type unsigned-integer;
+            }
+            field fabric-block-lost-packets {
+                sensor pipestats {
+                    path /components/component/integrated-circuit/pipeline-counters/drop/fabric-block/state/lost-packets;
+                }
+                type unsigned-integer;
+            }
+            field fabric-block-out-high-priority {
+                sensor pipestats {
+                    path /components/component/integrated-circuit/pipeline-counters/drop/fabric-block/state/out-high-priority;
+                }
+                type unsigned-integer;
+            }
+            field fabric-block-out-low-priority {
+                sensor pipestats {
+                    path /components/component/integrated-circuit/pipeline-counters/drop/fabric-block/state/out-low-priority;
+                }
+                type unsigned-integer;
+            }
+            field fabric-block-oversubscription {
+                sensor pipestats {
+                    path /components/component/integrated-circuit/pipeline-counters/drop/fabric-block/state/oversubscription;
+                }
+                type unsigned-integer;
```

Comment:
Are all these unsigned integers????

I could not find definition for these in the telemetry explorer. Can you please point me to the definitions of these sensors?

Reply:
Yes. Here you go:
http://ops.openconfig.net/branches/models/master/openconfig-platform.html#